javascript/javascript1/week4/getReply.js

```diff
@@ -0,0 +1,103 @@
+let userName = " ";
+let todo = [];
+function getReply(command) {
+  //'ask for name.'
+  const array = command.split(" ");
+  let response = " ";
+  if (command.includes("hello my name is ")) {
+    userName = array[array.length - 1];
+    return "Nice to meet you " + userName;
+  } else if (command.includes("what is my name")) {
+    return "your name is " + userName;
+  } else if (command.includes() {
+    return "your name is not find";
+  }
+  //'adding to todos.'
+  else if (command.includes("add ") && command.includes(" to my todos")) {
```

Comment:
Here again the problem is that you specify lowercase "add" whereas you are trying to log it out with "Add". Includes is unfortunately case-sensitive. Also you wrote "todo" on line 95 instead of "todos". 
(In a swapped scenario it would work though, because "todos" includes "todo", just not the other way around)